content/project-news/the-new-reactos-website.md

```diff
@@ -18,7 +18,7 @@ We added a direct link *Improve this page* to the bottom of every page.
 
 To better understand our philosophy, let's also explain what the new ReactOS website is not.  
 It is not another clickbait site that is constantly distracting you from the actual content and tracking your every move.
-In this age of ever-eroding privacy, the new ReactOS website wants to be a counterexample on how to build a website that protects your data instead of sharing it with third parties.
+In this age of ever-eroding privacy, the new ReactOS website wants to be an example on how to build a website that protects your data instead of sharing it with third parties.
```

Comment:
No, it's correct. It's counterexample to a plenty of sites that do not follow security and/or privacy guidelines. And it's age of ever-eroding privacy because of such sites.

Reply:
Counterexample is the example which disproves a proposition.
All the websites are sharing data with 3rd party. But, ReactOS doesn't. 
All the roses are red, except all those are not pink.
It's a way to opposite a general proposition, a universal fact or a theoretical enviroment.

Reply:
IMO, as a native English speaker... either one makes sense. 
Counterexample to me is just saying "hey we're showing an example here that breaks that normal and countering it with genuine privacy"
Example to me says "hey we want our website to be an example of how to properly treat one's privacy"